author.tf

```diff
@@ -505,6 +513,42 @@ module "author-dynamodb" {
   slack_alert_sns_arn = "${module.eq-alerting.slack_alert_sns_arn}"
 }
 
+resource "aws_elasticache_cluster" "memory-store" {
+  cluster_id                   = "${var.env}-redis"
+  engine                       = "redis"
+  node_type                    = "cache.t2.small"
```

Comment:
Is there any reason you went with small over micro?

Reply:
No particular reason. I'll switch to micro and see how it goes.

Reply:
It's slightly less memory so be worth seeing if we can get away with it, it shaves about £10 off monthly bill if micro is sufficient.

Reply:
Downgraded